syz-ci/syz-ci.go

```diff
@@ -144,6 +144,9 @@ type Config struct {
 	// Per-vm type JSON diffs that will be applied to every instace of the
 	// corresponding VM type.
 	PatchVMConfigs map[string]json.RawMessage `json:"patch_vm_configs"`
+	// Some commits don't live long. Cache them somewhere to increase lifetime.
+	// The cache is used by coverage merger.
+	GitArchive string `json:"git_archive"`
```

Comment:
We have syz-ci instances that test several different OSes (e.g. ci2 has netbsd). I think we don't want to push the whole NetBSD to the same git repo that archives Linux commits?

Reply:
PTAL